### PR TITLE
Rc 5.2.0 LabeledInput added new props

### PR DESCRIPTION
**Pull request by vik753**

### CODE REVIEW

File: uui-core/src/types/components/LabeledInput.tsx [hunk 1]
```diff
@@ -21,4 +19,14 @@ export interface LabeledInputCoreProps
...
      */
     htmlFor?: string;
+    /** A value from LabeledInput children */
+    value?: any;
+    /** Maximum text length, in characters */
+    maxLength?: number;
+    /** Showing current text length, in characters and maxLength */
+    charCounter?: boolean
+    /** Additional text in the bottom of LabeledInput */
+    footNote?: ReactNode;
```
Comment (AlekseyManetov) [inline uui-core/src/types/components/LabeledInput.tsx:29]: let name it without camelCase -  `footnote`
And change comment to:    /** Additional info positioned at the bottom of LabeledInput */

File: uui-core/src/types/components/LabeledInput.tsx [hunk 2]
```diff
@@ -21,4 +19,14 @@ export interface LabeledInputCoreProps
...
+    /** Additional text in the top of LabeledInput */
+    sideNote?: ReactNode;
```
Comment (AlekseyManetov) [inline uui-core/src/types/components/LabeledInput.tsx:31]: let name it without camelCase -  `sidenote`.
And fix comment to: /** Additional info positioned to the right side of label */

File: uui/components/widgets/StatusIndicator.tsx [hunk 3]
```diff
@@ -6,9 +6,13 @@ import css from './StatusIndicator.module.scss';
 export type StatusIndicatorColors = 'neutral' | 'info' | 'success' | 'warning' | 'critical';
 
 export interface StatusIndicatorProps extends IHasCX {
+    /** Component size. If omitted, 24 size will be used. */
     size?: '24' | '18' | '12';
+    /** Component color. */
     color?: StatusIndicatorColors;
+    /** Fill mode, solid is used by default */
     fill?: 'solid' | 'outline';
+    /** Component caption. It's required. */
```
Comment (AlekseyManetov) [inline uui/components/widgets/StatusIndicator.tsx:15]: Remove 'It's required.' because it's clear from property definition.

**Review by AlekseyManetov — CHANGES_REQUESTED** [review]

File: uui-core/src/types/components/LabeledInput.tsx (hunk 1 above)
Comment (vik753) [reply uui-core/src/types/components/LabeledInput.tsx:29]: fixed

File: uui-core/src/types/components/LabeledInput.tsx (hunk 2 above)
Comment (vik753) [reply uui-core/src/types/components/LabeledInput.tsx:31]: fixed

File: uui/components/widgets/StatusIndicator.tsx (hunk 3 above)
Comment (vik753) [reply uui/components/widgets/StatusIndicator.tsx:15]: fixed

**Review by siarhei-epam — APPROVED** [review]

**Pull request merged**